testcases/OpTestSTB.py

```diff
@@ -0,0 +1,623 @@
+        raise OpTestError(l_msg)
+
+    ##
+    # @brief This function collects CAPP loaded information from OPAL
+    # messagelog and if found determines whether it is 'loaded' or
+    # 'not loaded'.
+    # Returns CAPP_PAR_LOADED for CAPP partition loaded
+    # Returns CAPP_PAR_NOT_LOADED for CAPP partition not loaded
+    # Raises OpTestError when no information found
+    #
+    def capp_loaded(self):
+        l_capp_loaded = self.string_get(self.l_data,
+                                        BMC_CONST.CAPP_LOADED)
+        if l_capp_loaded:
+            l_capp_loaded_status = self.status_finder(l_capp_loaded,
+                                                      BMC_CONST.LOADED,
+                                                      BMC_CONST.NOT_LOADED)
+            if l_capp_loaded_status == BMC_CONST.LOADED:
+                print BMC_CONST.CAPP_PAR_LOADED
+                return BMC_CONST.CAPP_PAR_LOADED
+            elif l_capp_loaded_status == BMC_CONST.NOT_LOADED:
+                print BMC_CONST.CAPP_PAR_NOT_LOADED
+                return BMC_CONST.CAPP_PAR_NOT_LOADED
+            else:
+                print "CAPP partition loaded related error occured"
+                return BMC_CONST.FW_FAILED
+        else:
+            l_msg = "CAPP partition loaded details not seen"
+            raise OpTestError(l_msg)
+
+    ##
+    # @brief This function collects BOOTKERNEL loaded information from OPAL
+    # messagelog and if found determines whether it is 'loaded' or
+    # 'loaded'.
+    # Returns BOOTKERNEL_PAR_LOADED for BOOTKERNEL partition loaded
+    # Returns BOOTKERNEL_PAR_NOT_LOADED for BOOTKERNEL partition not loaded
+    # Raises OpTestError when no information found
+    #
+    def bootkernel_loaded(self):
+        l_bk_loaded = self.string_get(self.l_data,
+                                      BMC_CONST.BOOTKERNEL_LOADED)
+        if l_bk_loaded:
+            l_bk_loaded_status = self.status_finder(l_bk_loaded,
+                                                    BMC_CONST.LOADED,
+                                                    BMC_CONST.NOT_LOADED)
+            if l_bk_loaded_status == BMC_CONST.LOADED:
+                print BMC_CONST.BOOTKERNEL_PAR_LOADED
+                return BMC_CONST.BOOTKERNEL_PAR_LOADED
+            elif l_bk_loaded_status == BMC_CONST.NOT_LOADED:
+                print BMC_CONST.BOOTKERNEL_PAR_NOT_LOADED
+                return BMC_CONST.BOOTKERNEL_PAR_NOT_LOADED
+            else:
+                print "BOOTKERNEL partition loaded related error occured"
+                return BMC_CONST.FW_FAILED
+        else:
+            l_msg = "BOOTKERNEL partition loaded details not seen"
+            raise OpTestError(l_msg)
+
+    ##
+    # @brief This function checks for the device-tree node of Trusted boot
+    # Returns FW_SUCCESS when TB_DT_ENTRY found
+    # Returns FW_FAILED when TB_DT_ENTRY not found
+    #
+    def proc_tb_verify(self):
```

Comment:
We cannot verify the property path directly, as corresponding node path may change in P9.
change it to host_dt_prop_exists(self, i_node_path, i_prop) 
to check the property exists on that node path(for example node: /proc/device-tree/ibm,secureboot).  
and move the function to common/OpTestHost.py, we can call that function in testcase.